app/assets/javascripts/app/collections/notifications.js

```diff
@@ -0,0 +1,109 @@
+app.collections.Notifications = Backbone.Collection.extend({
+  model: app.models.Notification,
+  // URL parameter
+  /* eslint-disable camelcase */
+  url: Routes.notifications({per_page: 10, page: 1}),
+  /* eslint-enable camelcase */
+  page: 2,
+  perPage: 5,
+  unreadCount: 0,
+  unreadCountByType: {},
+
+  initialize: function() {
+    this.pollNotifications();
```

Comment:
It has been in production on diaspote for 3 months now with no problem. Maybe @Flaburgan can merge it on d-fr before it is merged on develop to see what happens?